src/main/kotlin/com/chattriggers/ctjs/minecraft/libs/renderer/Renderer.kt

```diff
@@ -775,47 +790,34 @@ object Renderer {
         renderBlackBox: Boolean = true,
         scale: Float = 1f,
         increase: Boolean = true,
+        visibleThroughWalls: Boolean = true,
     ) {
+        //#if MC<=11202
         var lScale = scale
         val renderPos = getRenderPos(x, y, z)
 
         if (increase) {
-            //#if MC<=11202
             val distance = sqrt(renderPos.x * renderPos.x + renderPos.y * renderPos.y + renderPos.z * renderPos.z)
-            //#else
-            //$$ val distance = sqrt(renderPos.x() * renderPos.x() + renderPos.y() * renderPos.y() + renderPos.z() * renderPos.z())
-            //#endif
-            val multiplier = distance / 120f //mobs only render ~120 blocks away
+            val multiplier = distance / 6f
```

Comment:
What does changing this from `120` mean

Reply:
Oh, well looking back, I actually missed something.  But these changes will make the scaling much more consistent between having `increase` true or false.

Before: increase true
![image](https://user-images.githubusercontent.com/46137467/180625534-caa08b98-909b-4984-95d0-72386283df69.png)


Before: increase false
![image](https://user-images.githubusercontent.com/46137467/180625531-14559275-c92d-4037-a82a-067c320cc3c2.png)


After: increase true
![image](https://user-images.githubusercontent.com/46137467/180625473-be48904b-a23a-425b-982b-14388c63069b.png)

After: increase false
![image](https://user-images.githubusercontent.com/46137467/180625481-ad84cdc9-abb6-46d4-b16d-71939911ff1f.png)